public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -0,0 +1,27 @@
+library pipes_examples.exponential_strength_pipe;
+
+import 'package:angular2/angular2.dart';
```

Comment:
sort imports. Always put `dart:` first

put a space between `dart:*` imports and `package:*` imports